tests/OrganizationsTest.php

```diff
@@ -96,6 +101,45 @@ public function testApplicationTokenWithScopeCanAccessOrganizationDetailAndProje
         $this->assertEquals($orgFromAdminToken, $orgFromAppToken);
```

Comment:
Upravy vyzeraju v teste v pohode ale vsimol som si troch riadkov ktorym moc nerozumiem 
```
$orgFromAppToken = $client->getOrganization($organization['id']);
$orgFromAdminToken = $client->getOrganization($organization['id']);
$this->assertEquals($orgFromAdminToken, $orgFromAppToken);
```
to bude nejaky omyl nie? prve dva riadky pouzivaju $client a volaju rovnaku metodu takze pak ten assert nedava zmysel.

Reply:
[91e2df0](https://github.com/keboola/kbc-manage-api-php-client/pull/295/commits/91e2df0b3b6efdaee077d8150b5a31373993446b)